Library/ViewModels/RewardCardContainerViewModelTests.swift

```diff
@@ -192,14 +192,14 @@ final class RewardCardContainerViewModelTests: TestCase {
     self.pledgeButtonEnabled.assertValues([true, true, true, true, false, false, false, true])
     self.pledgeButtonHidden.assertValues([false, false, false, false, false, false, false, false])
     self.pledgeButtonTitleText.assertValues([
-      "Pledge £6 or more",
-      "Pledge £6 or more",
-      "Pledge £6 or more",
-      "Pledge £6 or more",
+      "Select",
```

Comment:
Can we use `Strings.Select()`, ` Strings.No_longer_available()`, and     `Strings.Make_a_pledge_without_a_reward()` instead?

Reply:
So this is an interesting question. I realize we've been fairly inconsistent in tests with this but I'd like to actually do a sweep sometime to change every `String` assertion to be explicit rather than through `Strings.*`. I think the reason being that if the underlying string changes we'd actually want our tests to fail and for us to validate whether the test needs to be updated 🤔 

cc @Scollaco @ifbarrera for discussion.